README.md

```diff
@@ -25,6 +25,7 @@ BIDS currently supports the following data modalities with more to come in the f
 - behavioral
```

Comment:
```suggestion
- Behavioral
```

Reply:
As mentioned in https://github.com/bids-standard/bids-specification/pull/881#discussion_r747413582, we'll keep lower-case for behavioral, physiological AND microscopy to stay in scope for this PR and keep consistency. Changes to capital letters could be done in a separate PR.

Reply:
Sounds very good! Thanks!